app/league/configuration/page.tsx

```diff
@@ -0,0 +1,9 @@
+import LeagueConfigurationForm from "./form/form";
```

Comment:
nit: same as above, feels like we should have a new line

```suggestion
import LeagueConfigurationForm from "./form/form";

```